python/arcticdb/version_store/library.py

```diff
@@ -393,6 +393,11 @@ def __repr__(self):
         )
 
 
+class MergeStrategy(NamedTuple):
```

Comment:
What are the expected semantics of UPDATE vs INSERT? Just thinking of how this will be used, isn't `upsert` always what's wanted i.e. add missing, or update what's there?